scripts/test-init

```diff
@@ -12,6 +12,6 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 set -x
-make get-deps
-make govet
-make test
+make get-deps-init
+make govet-init
+make test-init
```

Comment:
just to be sure -- this isn't changing an existing script in a make target is it?
do we use the `scripts/test` in the `make test` target?

Reply:
This script is only used by the test-init dockerfile (which came from the init repo), which is only used by the test-in-docker-init target that I added to the Makefile from the init Makefile (and added the init suffix to).